pom.xml

```diff
@@ -236,13 +236,17 @@
         <version>0.13.3</version>
       </dependency>
 
+      <!-- Jackson libraries -->
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-core</artifactId>
+        <version>${jackson.version}</version>
+      </dependency>
       <dependency>
```

Comment:
we can import the jackson bom to cut down on all these dependencies. 

https://github.com/FasterXML/jackson-bom#usage